README.md

```diff
@@ -107,15 +107,78 @@ Running a build
 
 When everything is configured you can trigger a build with a push to the feedstock repo on github.
 
-Developing conda-smithy
+Developing `conda-smithy`
 -----------------------
 
-To develop conda smithy, use your favortite conda-based environment manager and create an environment based on the `environment.yml`.
+If you don’t already have a conda-based environment manager, install one. For example [Miniconda]((https://docs.anaconda.com/free/miniconda/miniconda-install/)).
 
-```
+To develop `conda-smithy`, use said environment manager and create an environment based on the `environment.yml`.
+
+```sh
 $ conda env create
 ```
 
+Then activate the environment, check that it works and install the dependencies:
+
+```sh
+$ conda activate conda-smithy
+$ which python
+# ☝️ should now point to an environment-specific python
+$ python -m pip install .
+```
+
+You might want to switch Python versions, for example if you wish to exactly replicate a CI issue locally. To do that:
+
+- Change the Python version in the `environment.yml`
+- Deactivate and delete old environment, if you have it: `conda deactivate && conda env delete -n conda-smithy`
+- (re-)run `conda env create` and activate the new environment with `conda activate conda-smithy`
```

Comment:
Not sure if this is the best way. Why not simply `conda install -n conda-smithy python=x.y`?